### ISSUE TITLE
a11y: Semantics Tree doesn't update fast enough for scrolling

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#11204]
On the Gallery App homepage in Android, select an item in the list of demos and start swiping right multiple times really fast. Continue swiping right when the end of the list is reached. Android will scroll and attempt to focus the next item in the list. However, the Semantics Tree has not been updated yet, focusing the next item fails (because from Android's a11y perspective there is no element) and the boing sound is played. After a couple of swipes, the semantic tree has been updated and everything works as expected.

**Comment by goderbauer** [comment]
This happens in release mode and debug mode. Tested on a Nexus 6P.

/cc @Hixie

**Comment by goderbauer** [comment]
I've tried this again on a Nexus 6P and on a Moto G4. There doesn't seem to be an issue anymore. Closing this for now.
